test-suite/testkit/Cargo.toml

```diff
@@ -21,6 +21,7 @@ travis-ci = { repository = "exonum/exonum" }
 [dependencies]
 actix = "0.7.9"
 actix-web = { version = "0.7.18", default-features = false }
+chrono = { version = "0.4.6" }
```

Comment:
Nit: Why isn't a short form (`chrono = "0.4.6"`) used here?

Reply:
Because it's in this form for main `exonum` `Cargo.toml`. I think we should revise our `Cargo.toml`'s and fix all issues like this at once.